impl/no-deps-transformer/src/main/java/org/wildfly/transformer/nodeps/TransformerImpl.java

```diff
@@ -167,7 +162,7 @@ private String replacePackageName(final String resourceName, final boolean dotFo
                     if (patches == null) {
                         patches = new ArrayList<>(countUtf8Items(clazz, constantPool));
```

Comment:
What is the JVM spec term used to describe the patch[] storage?  Is it okay if I look in the JVM 13 SPEC or do I have to look in JVM 8 spec?

I see that patch is array of int, what is the difference between patch[0] + patch[1].  From comments above getPatch(), I see that patch[0] is index of CONSTANT_Utf8_info and patch[1] difference in bytes after applying patches.  Are the comments above getPatch correct?

Reply:
Patch structure is just internal structure of "no dependencies transformer" @scottmarlow .
It has nothing to do with either JVM specification or Java language specification.
Yes, the comments above getPatch() method are correct and they describe the implementation detail.